nexus-sdk-js/src/ACL.ts

```diff
@@ -1,19 +1,34 @@
-export type Permission = 'read' | 'write' | 'own';
+export type PermissionsEndpointResponse = {
```

Comment:
I guess we should stick with Interface when describing an object. Types are good for union (which is why I used it before)